### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#676] [Galaxy Trivia] [answered]
Using plus (+), minus (-) and multiplication (x) operator signs. The three vertical and three horizontal equations must be true.

<img width="202" alt="image" src="https://user-images.githubusercontent.com/5120290/236083412-311a2d24-b5e1-4305-b6e3-8f02ad662fdb.png">

**Comment by peterbaccaro** [comment, marked as answer]
![puzzle](https://user-images.githubusercontent.com/74666079/236123937-580a74c7-294a-4590-acfa-9d769dbae1fc.png)
